### PR TITLE
Added examples of preceding LOAD and some more

### PR DESCRIPTION
**Pull request by janmiderback**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -42,11 +42,13 @@ examples are located in the [scripts/](./scripts/) folder.
...
 ----------- | ----------
-[load-csv-file](./scripts/load-csv-file) | Loading data from a CSV file.
-[load-xlsx-file](./scripts/load-xlsx-file) | Loading data from an Excel file.
-[load-inline-table](./scripts/load-inline-table) | Loading an inline table 
-[load-subset-of-fields](./scripts/load-subset-of-fields) | Loading a subset of fields.
-[rename-fields](./scripts/rename-fields) | Renaming fields.
-[load-field-functions](./scripts/load-field-functions)
-[concat-tables-auto](./scripts/concat-tables) | Automatic concatenation into one table.
-[concat-tables-forced](./scripts/concat-tables) | Forced concatenation into one table.
+[load-csv-file](./scripts/load-csv-file) | <ul><li>Loading data from a CSV file.</ul>
```
Comment (janmiderback) [inline README.md]: Do lists render ok? Let's check. If not, I'll revert.

Comment (hrigner) [reply README.md]: Yes they do :)

**Review by hrigner — APPROVED** [review]
LGTM! However, it actually took me a while to understand, but I don't know how to simplify it..

**Comment by janmiderback** [comment]
@hrigner Good point. I also struggled with simplifying. I'll check what I can do.

**Comment by hrigner** [comment]
Maybe add "Simplifying your script by reusing calculations"? I didn't really get why one should do a preceding load. Or well, you kind of already have that..

File: scripts/preceding-load-reuse-calc [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+// By using preceding LOAD, we can reuse a calculation and avoid the double
+// calculation by adding an extra LOAD statement.
+
+EvenOrOddness:
+    LOAD
+        Id AS KeyId,
+        IF (Even(Id), 'Even', 'Odd') AS EvenOrOdd;
+    LOAD
+        iterno() - 1 + Base AS Id
+        WHILE iterno() <= Length;
```
Comment (StefanEnberg) [inline scripts/preceding-load-reuse-calc]: We should camelcase iterno() => IterNo() to be consistent with the documentation https://help.qlik.com/sv-SE/qlikview/November2017/Subsystems/Client/Content/Scripting/CounterFunctions/IterNo.htm

File: scripts/preceding-load-calc-field [hunk 3]
```diff
@@ -0,0 +1,6 @@
+// The load script below shows how to use preceding LOAD to calculate a new
+// field, Tax, in one table in a single pass.
+
+Taxes:
+    LOAD *, Salary * 0.10 as Tax;
```
Comment (StefanEnberg) [inline scripts/preceding-load-calc-field]: Maybe extract 0.10 to a variable like SalesTax to make it cleaner and more descriptive

**Review by StefanEnberg — APPROVED** [review]
Minor comments but otherwise LGTM

**Comment by StefanEnberg** [comment]
The changes looks good

**Comment by janmiderback** [comment]
Closes #5

**Pull request merged**